Formula/ceu.rb

```diff
@@ -0,0 +1,31 @@
+class Ceu < Formula
+  desc "Reactive language alternative to C"
+  homepage "http://www.ceu-lang.org/"
+  url "https://github.com/fsantanna/ceu.git", :tag => "v0.30"
+  depends_on "lua"
+
+  resource "lpeg" do
+    url "https://luarocks.org/manifests/gvvaughan/lpeg-1.0.1-1.src.rock", :using => :nounzip
+    sha256 "149be31e0155c4694f77ea7264d9b398dd134eca0d00ff03358d91a6cfb2ea9d"
+  end
+
+  def install
+    luapath = libexec/"vendor"
+    ENV["LUA_PATH"] = "#{luapath}/share/lua/5.3/?.lua"
+    ENV["LUA_CPATH"] = "#{luapath}/lib/lua/5.3/?.so"
+
+    resource("lpeg").stage do
+      system "luarocks", "build", "lpeg", "--tree=#{luapath}"
+    end
+
+    system "make"
+    bin.install "src/lua/ceu"
+
+    env = { :LUA_PATH => ENV["LUA_PATH"], :LUA_CPATH => ENV["LUA_CPATH"] }
+    bin.env_script_all_files(libexec/"bin", env)
+  end
+
+  test do
+    system "#{bin}/ceu", "--version"
```

Comment:
The test needs to exercise and verify the functionality of the software. Something that builds a Hello World for Ceu would be good